src/index.js

```diff
@@ -44,24 +44,19 @@ module.exports = class BlockService {
   }
 
   putStream () {
-    let ps
     if (this.isOnline()) {
-      // NOTE: This will have to change in order for bitswap
-      // to understand CID
-      ps = this._bitswap.putStream()
+      return this._bitswap.putStream()
     } else {
-      ps = this._repo.blockstore.putStream()
+      return pull(
+        pull.map((blockAndCID) => {
```

Comment:
Wait, BlockService is blockAndCID and now bitswap is also blockAndCID, because bitswap needs to know CIDs, where do we in Bitswap move it to just Block?